omnibus/files/private-chef-cookbooks/private-chef/recipes/fix_permissions.rb

```diff
@@ -24,3 +24,14 @@
 execute "find #{GEM_PATH} -perm /u=r,g=r,o=r ! -perm /u=x -exec chmod 644 {} \\;" do
   user 'root'
 end
+
+[
+  '/opt/opscode/embedded/nginx',
+  "#{node['private_chef']['nginx']['dir']}",
```

Comment:
Are nginx directories exist while doing this operation. As per my testing It was failing.
Reference - https://buildkite.com/chef/chef-chef-server-main-omnibus-adhoc/builds/3076#1186b387-f85f-4d90-90ce-df5f951503ed
![Screenshot 2021-08-25 at 2 07 20 PM](https://user-images.githubusercontent.com/76031665/130757738-6391da30-16bc-47a3-b031-88db47362da2.png)

Reply:
I had not seen that but had only configured after deployed, not from initial install.  I updated the execute resource in fix_permissions with a guard, which it should have had to begin with, which corrects this.